modules/cpdos/hbh.py

```diff
@@ -148,12 +144,7 @@ def HBH(
                     )
 
             except requests.exceptions.ConnectionError as e:
-                pass
-                #logger.exception(e)
-
-            except Exception as e:
-                pass
-                #logger.exception(e)
+                logger.exception(e)
```

Comment:
Removed Exception to avoid swallowing KeyInterrupt and bubble up any non requests exceptions